backend/app/router/endpoints/notes.py

```
@@ -0,0 +1,252 @@
+from fastapi import APIRouter, HTTPException
+from typing import List
+import os
+from app.pydantic_models.notes import Note
+
+router = APIRouter()
+
+NOTES_DIR = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), 'notes')
```

Pylint (3.3.7)

[convention] Line too long (111/100)

(C0301)